### DISCUSSION TITLE
Theres no update for android?

### DISCUSSION THREAD

**Discussion by nobidebian** [ppy/osu#26838] [Q&A] [answered]
hello i just want to know if theres no update on android, because theres no apk file on the releases section, and when im launching osu, there is a notifcation saying that theres a new version of osu, but yea i can't find it on the release section. or is there any other way to update?

**Comment by Speed-sters** [comment, marked as answer]
https://github.com/ppy/osu/issues/26809

There are major issues with it right now, and it's been pulled for the time being.
